src/TLuaInterpreter.cpp

```diff
@@ -11321,50 +11321,59 @@ void TLuaInterpreter::setAtcpTable(const QString& var, const QString& arg)
     host.raiseEvent(event);
 }
 
-// No documentation available in wiki - internal function
-void TLuaInterpreter::setGMCPTable(QString& key, const QString& string_data)
+void TLuaInterpreter::raiseProtocolEvent(
+    QString& key,
+    const QString& string_data,
+    const QString& protocol,
+    const QStringList& tokenList)
```

Comment:
Have you got access to clang formatter? We have `.clang-format` file at the root.